### PR TITLE
Add console to cbgrid

### PR DESCRIPTION
**Pull request by grimpy**
See: #1705

Signed-off-by: Jo De Boeck <deboeck.jo@gmail.com>

### CODE REVIEW

File: apps/cbportal/base/CBGrid/Virtual Machines/VMConsole.wiki [hunk 1]
```diff
@@ -0,0 +1,21 @@
+@usedefaults
+{{vmachine}}
+h2. Virtual Machine: ${name}
+{% if status != "DESTROYED" %}
```
Comment (alichaddad) [inline apps/cbportal/base/CBGrid/Virtual Machines/VMConsole.wiki]: Should probably check for deleted as well

File: apps/cbportal/base/CBGrid/Virtual Machines/VMConsole.wiki [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+{{include:vmachine_actions}}
+
+{% endif %}
+{% if status == 'RUNNING' %}
+{% if accounts -%}
+{{html:
+<strong>Login:</strong> ${accounts[0].login}
+<a href="javascript:sendText('${accounts[0].login}');"><i class="glyphicon glyphicon-paste" title="Copy password to console"></i></a>
```
Comment (alichaddad) [inline apps/cbportal/base/CBGrid/Virtual Machines/VMConsole.wiki]: Copy login not password

Comment (grimpy) [reply apps/cbportal/base/CBGrid/Virtual Machines/VMConsole.wiki]: Fixed

File: apps/cbportal/base/CBGrid/Virtual Machines/VMConsole.wiki (hunk 1 above)
Comment (grimpy) [reply apps/cbportal/base/CBGrid/Virtual Machines/VMConsole.wiki]: This is required for the restore action

**Pull request merged**